kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java

```diff
@@ -16,16 +16,47 @@
  * code into multiple, classes.
  *
  * <p>TODO: Have a read only chunk info for read only chunks so we don't accidentally update it.
- * TODO: Add a state machine for a chunk?
+ *
+ * <p>TODO: Add a minOffset field to chunk and snapshot metadata.
  */
 public class ChunkInfo {
+  // Use a future time of Dec 31, 3000 instead of INSTANT.MAX since it overflows.
+  public static final long MAX_FUTURE_TIME = Instant.ofEpochSecond(32535158400L).toEpochMilli();
```

Comment:
Ah yeah - since we're converting it to milliseconds that would cause an overflow - `Instant.MAX` uses the max long in seconds. In that case we should just use the `Long.MAX_VALUE`.